codec-http/src/main/java/io/netty/handler/codec/http/HttpMethod.java

```diff
@@ -143,18 +133,14 @@ private HttpMethod(String name, boolean bytes) {
             }
         }
 
-        this.name = name;
-        if (bytes) {
-            this.bytes = name.getBytes(CharsetUtil.US_ASCII);
-        } else {
-            this.bytes = null;
-        }
+        this.name = new AsciiString(name);
+        nameAsString = name;
     }
 
     /**
-     * Returns the name of this getMethod.
+     * Returns the name of this method.
      */
-    public String name() {
+    public AsciiString name() {
```

Comment:
It would be nice to have a clarified stance in Netty for the deprecation policy (is it currently defined anywhere).  Not being able to change 4.1 creates a bit of merge/maintenance "hell" (as described in http://netty.io/news/2014/12/17/changes-in-deprecation-policy.html) between 4.0, 4.1, master.  This specific change is small but some others (i.e. headers) may be more complex.